extensions/ql-vscode/src/data-extensions-editor/data-extensions-editor-view.ts

```diff
@@ -75,6 +86,17 @@ export class DataExtensionsEditorView extends AbstractWebview<
     await this.loadExternalApiUsages();
   }
 
+  protected async saveYaml(yaml: string): Promise<void> {
+    const modelFilename = this.modelFileName;
```

Comment:
This tripped me up because I wasn't expecting the getter to be doing so much work - it just looks like we're accessing an instance field. What do you think about turning the getter into a normal function so that it's more obvious what's going on? E.g. `calculateModelFileName` (or something like that).